quickstatements/latest/php.ini

```diff
@@ -0,0 +1 @@
+date.timezone = "UTC"
```

Comment:
evil hardcoded?

Reply:
Perhaps. I'll make it configurable. This is really only there because without it php spits the warns into the JSON api responses from QS and break parsing by everything.